test/unit/objects/test_prefixtable.py

```diff
@@ -1,62 +0,0 @@
-import numpy as np
```

Comment:
As Corey mentioned, keep this file for now to avoid merge issues with his branch.

Reply:
Done